integration_test/gce/gce_testing.go

```diff
@@ -604,15 +618,47 @@ func runRemotelyWindows(ctx context.Context, logger *log.Logger, vm *VM, command
 	if err != nil {
 		return CommandOutput{}, err
 	}
-	return runCommand(ctx, logger, "",
+	output, exitCode, err := runCommand(ctx, "",
 		[]string{winRM(),
 			"--host=" + vm.IPAddress,
 			"--username=" + vm.WindowsCredentials.Username,
 			"--password=" + vm.WindowsCredentials.Password,
-			fmt.Sprintf("--command=powershell -NonInteractive -encodedcommand %q", base64.StdEncoding.EncodeToString([]byte(encoded))),
+			fmt.Sprintf("--command=powershell -NonInteractive -OutputFormat Text -EncodedCommand %q", base64.StdEncoding.EncodeToString([]byte(encoded))),
```

Comment:
adding -OutputFormat Text does nothing, but I included it here because:

1. it's always bothered me how this seemingly-useful option is not called out in our code anywhere
2. if microsoft ever fixes powershell's behavior, we'll start using the fix automatically.